README.md

```diff
@@ -106,6 +106,20 @@ example. The purpose is not to compare performance, but to show the
 similarities and differences between them. See block_host for the OpenMP 
 implementation.
 
+### SYCL-Graph Samples
+
+Code examples demonstrating the usage of [`sycl_ext_oneapi_graph`](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc) extension.
```

Comment:
More of a question for the repo maintainers than Konrad, but functionalities from the different tests will be available in different oneAPI releases. Does that need documented here?